pd/migrations/20211216025603_init.sql

```diff
@@ -33,12 +33,24 @@ CREATE INDEX ON notes (position);
 CREATE INDEX ON notes (height);
 
 CREATE TABLE IF NOT EXISTS validators (
-    tm_pubkey bytea NOT NULL PRIMARY KEY,
-    voting_power bigint NOT NULL
+    tm_pubkey bytea NOT NULL PRIMARY KEY
 );
 
 CREATE TABLE IF NOT EXISTS validator_fundingstreams (
     tm_pubkey bytea NOT NULL,
     address varchar NOT NULL,
     rate_bps bigint NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS base_rates (
+    epoch bigint PRIMARY KEY,
+    base_rate bigint NOT NULL
```

Comment:
passive note/non-blocking: I think postgres's `bigint` type is actually an i64, our rates are specified as being `u64`, so we'd have to do a type conversion here when we pull these out of the database and check for the overflow condition. This shouldn't be an issue since our rates are specified with an implicit 10^8 denominator, the u64 typing is so that we can safely compute products of rates in the same type without having to worry about overflow.